frontend/views/components/banners/BannerGeneral.vue

```diff
@@ -7,19 +7,17 @@
       aria-live='polite'
     )
       i(:class='`icon-${ephemeral.icon} is-prefix`')
-      render-message-with-markdown(:text='ephemeral.message')
+      .c-banner-content(v-safe-html:a='ephemeral.message')
 </template>
 
 <script>
 import TransitionExpand from '@components/TransitionExpand.vue'
 import { debounce } from '@model/contracts/shared/giLodash.js'
-import RenderMessageWithMarkdown from '@containers/chatroom/chat-mentions/RenderMessageWithMarkdown.js'
```

Comment:
Using `RenderMessageWithMarkdown.vue` here is wrong but regardless, it internally used `v-safe-html` custom directive which is the thing we need here. So I just replaced this component with a `<div />` that uses this directive.